controller/controller.js

```diff
@@ -11,6 +12,27 @@ const getTopics = (req,res,next) => {
     
 }
 
+const getApis = (req, res) => {
+    const documentation = {};
+  
+    for (const endpoint in endpointsJSON) {
+      documentation[endpoint] = {
+        description: endpointsJSON[endpoint].description,
+        queries: endpointsJSON[endpoint].queries || [],
+        exampleResponse: endpointsJSON[endpoint].exampleResponse || {},
+      };
+    }
+  
+    res.status(200).json(documentation);
+  };
 
+  const getArticleById = (req, res, next) => {
+    const { article_id } = req.params;
+    readArticleById(article_id)
+      .then((article) => {
+        res.status(200).json({ article });
+      })
+      .catch(next);
```

Comment:
Great controller! That's a lovely syntax for the catch using `.catch(next)` - but then it doesn't look like anything is happening with that error, how should we handle the error in the _app.js_ ?